frontend/app/src/index.tsx

```diff
@@ -13,7 +13,7 @@ import AddAmbassador from './scenes/Home/scenes/Ambassadors/scenes/AddAmbassador
 import { SingleRoute } from './interfaces';
 import Categories from './scenes/Home/scenes/Category';
 import UpsertItem from './scenes/Home/scenes/UpsertItem';
-import AddCategory from './scenes/Home/scenes/AddCategory';
+import AddCategory from './scenes/Home/scenes/UpsertCategory';
```

Comment:
import it as `UpsertCategory`

Reply:
Sure!

Reply:
Resolved with commit [7d2ab6e](https://github.com/sef-global/sef-core/pull/81/commits/7d2ab6e714f90975ff227736c6d722760c7c27be)